CHANGELOG.md

```diff
@@ -0,0 +1,102 @@
+# Visual F<span>#</span>
+All notable changes to this project will be documented in this file.
+
+This project **don't** uses [Semantic Versioning](http://semver.org/)
```

Comment:
Please update it to:
This project **doesn't** use [Semantic Versioning](http://semver.org/)